services.yaml

```diff
@@ -3218,6 +3283,52 @@ definitions:
         items:
           type: string
           example: MyApp
+  PublishMetricsSettings:
+    type: object
+    properties:
+      enable:
+        type: boolean
+        example: true
+      server:
```

Comment:
it's actually `host:port` so i'm not sure "server" is a good term

Reply:
updated + added description to this field